sd/StringParser/src/com/company/StringFilter.java

```diff
@@ -0,0 +1,33 @@
+package com.company;
+import java.util.*;
+
+public class StringFilter {
+
+    public static String filter(String inputText, HashSet<String> blackList)
```

Comment:
inputText is input string or input file name?

Reply:
I didn't even think about that! good catch, refactored variable name to inputString